packages/core/package.json

```diff
@@ -3,18 +3,22 @@
   "version": "0.3.0",
   "description": "Runtime-agnostic core of ESLint",
   "type": "module",
-  "types": "./src/types.ts",
+  "types": "./dist/esm/types.d.ts",
+  "exports": {
+    "types": {
+      "import": "./dist/esm/types.d.ts",
+      "require": "./dist/cjs/types.d.cts"
+    }
+  },
   "files": [
-    "src"
+    "dist"
   ],
   "publishConfig": {
     "access": "public"
   },
-  "directories": {
-    "test": "tests"
-  },
```

Comment:
There is no "tests" directory in this package. We will want to add it at some point.